TGC/common/governments.txt

```diff
@@ -193,6 +193,7 @@ theocracy = {
 
 	election = no
 	appoint_ruling_party = yes
+	flagType = theocracy
```

Comment:
You cannot add a flagType without including a placeholder theocracy flag for *ALL* tags.

Reply:
Yes I know, I still have to finish adding the theocracy flags but I committed what I did yesterday so if anything wrong happens I don't have to restart from scratch